tests/test_errors.py

```diff
@@ -333,6 +333,7 @@ def test_i_error():
     with pytest.raises(FactorEvaluationError):
         feols("Y ~ i(f1, X1, ref=a)", data)
 
+
```

Comment:
Sorry, pre-commit does not seem to be set up correctly on my local machine.

Reply:
all good! :)